cloud/google/gceproviderconfig/v1alpha1/types.go

```diff
@@ -29,5 +29,15 @@ type GCEProviderConfig struct {
 	MachineType string `json:"machineType"`
 
 	// The name of the OS to be installed on the machine.
-	OS string `json:"os"`
+	OS    string `json:"os"`
+	Disks []Disk `json:"disks"`
+}
+
+type Disk struct {
+	InitializeParams DiskInitializeParams `json:"initializeParams"`
```

Comment:
Why add this level of indirection?  Wouldn't
```yaml
disks:
  - diskSizeGb: 30
    diskType: "pd-standard"
  - diskSizeGb: 50
    diskType: "pd-standard"
```

work?

Reply:
It would be then it would not match the GCE API / structure which is I thought what we wanted to do. It has the InitializeParams structure. 

https://cloud.google.com/compute/docs/reference/rest/beta/instances/insert

Do agree what you are proposing is fine too -- let me know if you want to move to that structure.